.github/workflows/ci.yaml

```diff
@@ -0,0 +1,67 @@
+name: CI
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+            - develop
+
+jobs:
+    tests:
+        name: Tests
+        runs-on: ${{ matrix.os }}
+        strategy:
+            fail-fast: false
+            matrix:
+                os:
+                    - ubuntu-latest
+                    - windows-latest
+                php:
+                    - '7.2'
+                    - '7.3'
+                    - '7.4'
+#                    - '8.0'
```

Comment:
You can remove this because we will never support PHP 8 on `2.x` due to its maintenance status and add `7.1` instead which is the minimum supported version